hyper-schema.json

```diff
@@ -47,52 +47,64 @@
                 },
                 "schema": {
                     "description": "Schema describing the data to submit along with the request",
-                    "allOf": [ { "$ref": "#" } ]
+                    "allOf": [ { "$ref": "#/definitions/subSchema" } ]
                 }
             }
+        },
+        "subSchema": {
+            "oneOf": [
+                {
+                    "allOf": [
+                        { "$ref": "#" },
+                        { "$ref": "http://json-schema.org/draft/schema#/definitions/notJsonReference" }
```

Comment:
Logged #182 (boolean schemas, because #167 went in without meta-schema change because I thought this would go in first) and #138 (for the relative reference thing).